pyecoregen/templates/module_utilities.tpl

```diff
@@ -1,5 +1,16 @@
 {%- macro generate_enum(e) %}
-{{ e.name }} = EEnum('{{ e.name }}', literals=[{{ e.eLiterals | map(attribute='name') | map('pyquotesingle') | join(', ') }}])
+{% set name = e.name %}
+{% if textX %}
```

Comment:
Same as before, I'm not very sure I got it about the `EEnum`, but perhaps I missed something. If you have a small example, it would be great (currently, I only tested on the `library.ecore` metamodel).